templates/base.tx

```diff
@@ -131,12 +131,27 @@
   </header>
   <!--end header-->
 
+% if $feedback_banner {
+  <div class="govuk-body" id="chs-customer-insights-top">
+   <div class="govuk-width-container">
+     <a href="http://resources.companieshouse.gov.uk/csat/index.shtml?Service=CHS">Tell us what you think of Companies House</a>
+   </div>
+ </div>
+% }
+
   <main id="page-container" class="search <% $classes %>" role="main">
 
-        <div class="phase-banner">
-	    <div class="small-text">Companies House does not verify the accuracy of the information filed
-            </div>
-        </div>
+  % if $feedback_banner {
+    <div>
+      <div class="small-text">Companies House does not verify the accuracy of the information filed</div>
+    </div>
+  % }
+
+  % if !$feedback_banner {
+    <div class="phase-banner">
```

Comment:
Can we just make the class conditional based on `feedback_banner` to save duplicating this div twice?